docs_src/design/ucr/0002-Device-Parent-Child-Relationships.md

```diff
@@ -0,0 +1,109 @@
+
+The sound-bound Device Service that creates a Device is ideally the service which establishes this relationship data, though it is possible that it is unaware of the parent-child relationship. It should be permitted, therefore, for this relationship information to also be set by north-bound services (most likely the UI) and simply ignored by the south-bound Device Service.
+
+A potential solution might add a field to the device structure such as
+```json
+"relationships": [
+        "parent": "First-Floor-Gateway"
+      ]
+```
+It is probably also necessary to indicate which device is the "main" or "publisher" device (ie, the gateway device), 
+as any devices without a configured relationship will probably be inferred to be children of that device.
+
+### Extensions to the main Use Case
+1. In addition to parent-child relationships, other relationships might be indicated in this metadata, such as an
+"extends" when an analytic service extends an existing south-bound device, adding new Device Resources beyond what the south-bound service provides.
+2. Some services add "devices" which have no physical counterpart, eg, for an NTP client service where the "device" 
+serves simply as a container for the Resources necessary to configure and report the status of the service.
+In these cases, it would be helpful for the other services if it described itself as something like a "system" device,
+meaning one that doesn't have a physical (south-bound or hardware) counterpart.
+3. It could be possible for north-bound or analytic services to add metadata of their choice, related to an individual device, in this "relationships" area. 
+This definitely muddies the clear objective of this UCR, but consideration of it might steer the ultimate solution. 
+As application service developers, we have felt at times the desire to "annotate" a device with something more formal than just another label; 
+if other reviewers agree with this notion, it can be adopted here, since it follows the same lines 
+(metadata that other services might add to a device).
+
+
+### Existing solutions
+The Device structure in Eaton's legacy products indicated this parent-child relationship bidirectionally: each device indicated its parent device (if any) with one field, and its child devices (if any) with a list of IDs.
+
+The Device structure in Eaton's cloud solution is a "DeviceTree", which is a recursive, hierarchial structure of the connected devices, starting with the "publisher" device and its first-level child devices.
+
+There is the BACnet "virtual routed devices" model, but I would not recommend it, as it is too artificial for a simple relationship.
+
+The existing EdgeX UIs group devices by their Device Service, which is a good approach for simple devices without children of their own, but fails if those devices have child devices too.
+
+### Requirements
+No additional requirements.
```

Comment:
There might some requirements about how to query or update the devices with relationships

Reply:
There need to be some requirements here. Something like:
1) Ability to identify a device's parent device
2) Ability to identify a device's child devices.

Reply:
I mis-read the template; thought it only wanted "additional" requirements here. Yes, a summary of requirements makes sense; will do.